docs/ota-client-guide/modules/ROOT/pages/setup-boot-image-for-ostree.adoc

```diff
@@ -0,0 +1,42 @@
+= Set up boot image layout for OSTree compatibility
+ifdef::env-github[]
+
+[NOTE]
+====
+We recommend that you link:https://docs.ota.here.com/ota-client/latest/{docname}.html[view this article in our documentation portal]. Not all of our articles render correctly in GitHub.
+====
+endif::[]
+
+This is the second step in getting a new board running after xref:add-board-class.adoc[adding a new board class for the new target board in meta-updater].
+
+This step involves getting Yocto to build a physical image with the partitions in the correct place, with the necessary modifications for OSTree. The standard approach here is to use https://www.yoctoproject.org/docs/2.7/dev-manual/dev-manual.html#creating-partitioned-images-using-wic[Wic, the OpenEmbedded Image Creator].

```

Comment:
Currently, we still recommend thud (2.6) as the default Yocto release branch, so we should point at the corresponding docs. We should consider defining that version (or the whole base URL) once and reusing it to reduce the amount of updating we have to do. And on a related note, do we have a link checker in CI somewhere?
```suggestion
This step involves getting Yocto to build a physical image with the partitions in the correct place, with the necessary modifications for OSTree. The standard approach here is to use https://www.yoctoproject.org/docs/2.6/dev-manual/dev-manual.html#creating-partitioned-images-using-wic[Wic, the OpenEmbedded Image Creator].
```

Reply:
We do have a link checker. It runs on PR branches from the main repo, and also runs once a week on the live site as a scheduled task. I haven't yet figured out how to run gitlab CI when a github PR is opened, though--do you have a solution for that?

Reply:
> I haven't yet figured out how to run gitlab CI when a github PR is opened, though--do you have a solution for that?

Yes. In our case, we mirror our github repos on gitlab and our gitlab configuration includes some logic to run on most PRs. I forget exactly how that last part is specified but for aktualizr it should be either in `ci/gitlab/.gitlab-ci.yml` or in the gitlab settings. I can look into it if it isn't easy to find.

Reply:
Currently, we still recommend thud (2.6) as the default Yocto release branch, so we should point at the corresponding docs. We should consider defining that version (or the whole base URL) once and reusing it to reduce the amount of updating we have to do.

Yup, agreed. @ammaboateng, the best way to do this would be to add an attribute to [aktualizr-version.adoc](https://github.com/advancedtelematic/aktualizr/blob/master/docs/ota-client-guide/modules/ROOT/pages/_partials/aktualizr-version.adoc), then `include::[]` that file in this one. Message me if you have any trouble.